### ISSUE TITLE
Deleting a frame via hotkey Delete, deletes the node

### ISSUE THREAD

**Issue by TheYellowArchitect** [godotengine/godot#66487]
### Godot version

4.0 beta1

### System information

Artix (Arch Linux without systemd), Vulkan via NVIDIA official drivers

### Issue description

I am doing this official tutorial https://docs.godotengine.org/en/latest/getting_started/first_2d_game/02.player_scene.html

I notice I put wrong sprite, so I select it (click) 
![godot-issue1](https://user-images.githubusercontent.com/25630803/192491810-e0fe5681-ba49-4cdd-b28b-e7886a1ac373.png)

And press Delete, expecting to just delete the sprite. But it deletes the entire animation node.
![godot-issue2](https://user-images.githubusercontent.com/25630803/192499925-7822c839-63f3-4286-bf64-ec2fe82d2157.png)

It seems Shift+Delete deletes the AnimationSprite2d, so I think Delete should delete the frame if it's selected.

tl;dr: Negligible QoL bug

### Steps to reproduce

Mentioned above

**Comment by KoBeWi** [comment]
Duplicate of #63804
